src/app/navigation/actions.ts

```diff
@@ -1,8 +1,15 @@
+import queryString from 'query-string';
 import { createStandardAction } from 'typesafe-actions';
 import { AnyHistoryAction, AnyMatch, HistoryAction, LocationChange } from './types';
 
 export const callHistoryMethod = createStandardAction('Navigation/callHistoryMethod')<AnyHistoryAction>();
-export const locationChange = createStandardAction('Navigation/locationChange')<LocationChange>();
+export const locationChange = createStandardAction('Navigation/locationChange').map(
```

Comment:
it is dispatched in 20 files, 41 times - mostly test cases
Should I update it everywhere instead of using this `.map`?

Reply:
only one dispatch is outside of test files, but since the query and search are basically the same data, i suppose it seems a little weird to require passing both into the action. lets leave it the way you have it